zuul.d/project.yaml

```diff
@@ -4,4 +4,9 @@
     github-check:
       jobs:
         - openstack-k8s-operators-content-provider
```

Comment:
openstack-k8s-operators-content-provider shoudl have been removed in this test
as its replaced by openstack-meta-content-provider

Reply:
Done